forge/src/main/java/com/wynntils/forge/mixins/ForgeGuiMixin.java

```diff
@@ -22,24 +22,28 @@ protected ForgeGuiMixin(Minecraft arg) {
         super(arg, arg.getItemRenderer());
     }
 
-    @Inject(method = "render", at = @At("HEAD"))
-    private void onRenderGuiPre(PoseStack poseStack, float partialTick, CallbackInfo ci) {
+    @Inject(method = "render(Lnet/minecraft/client/gui/GuiGraphics;F)V", at = @At("HEAD"))
+    private void onRenderGuiPre(GuiGraphics guiGraphics, float partialTick, CallbackInfo ci) {
         MixinHelper.post(
-                new RenderEvent.Pre(poseStack, partialTick, this.minecraft.getWindow(), RenderEvent.ElementType.GUI));
+                new RenderEvent.Pre(guiGraphics, partialTick, this.minecraft.getWindow(), RenderEvent.ElementType.GUI));
     }
 
-    @Inject(method = "render", at = @At("RETURN"))
-    private void onRenderGuiPost(PoseStack poseStack, float partialTick, CallbackInfo ci) {
-        MixinHelper.post(
-                new RenderEvent.Post(poseStack, partialTick, this.minecraft.getWindow(), RenderEvent.ElementType.GUI));
+    @Inject(method = "render(Lnet/minecraft/client/gui/GuiGraphics;F)V", at = @At("RETURN"))
+    private void onRenderGuiPost(GuiGraphics guiGraphics, float partialTick, CallbackInfo ci) {
+        MixinHelper.post(new RenderEvent.Post(
+                guiGraphics, partialTick, this.minecraft.getWindow(), RenderEvent.ElementType.GUI));
     }
 
-    @Inject(method = "renderFood", at = @At("HEAD"), cancellable = true, remap = false)
-    private void onRenderFoodPre(int width, int height, PoseStack poseStack, CallbackInfo ci) {
+    @Inject(
+            method = "renderFood(IILnet/minecraft/client/gui/GuiGraphics;)V",
+            at = @At("HEAD"),
+            cancellable = true,
+            remap = false)
```

Comment:
I see this is not changed now, but do you know why this and the next method is remap = false?

Reply:
Yes. These are forge mixins, so most of the time we are mixing into forge added methods. These are not obfuscated, there is no need to map to intermediary (or to any other mapping).

Reply:
Ah, so some of them are mixing into vanilla methods, but only needed on forge, and other are mixing into forge methods?

Maybe we should acknowledge the later with a comment?

Reply:
Forge mixins only mixin into forge methods since the common mixins don't work on forge. Hence, a case for a forge mixin to mixin into not-forge code is really low. And the game doesn't compile without correct settings. So I think this is fine without explicit comments.